lib/src/authenticate/azure_adb2c_user.dart

```diff
@@ -18,6 +21,10 @@ class AzureADB2CUserSession extends UserSession {
       'receiveTimeout': config['receiveTimeout'],
     });
     _azureKey = config['azureKey'];
+    _azureSecret = config['azureSecret'];
+    _azureSubject = config['azure_subject'];
```

Comment:
Rename key to azureSubject. Check for other keys

Reply:
also add default empty value (prepare for nullsafety later)

Reply:
Done